### ISSUE TITLE
False positive in BracesForMethod rule

### ISSUE THREAD

**Issue by erdi** [CodeNarc/CodeNarc#376]
If there is a line comment between a method signature and an annotation for that method then the rule reports a violation when it shouldn't. Example code triggering the false positive:

```
@AnAnnotation
// some comment
void aMethod() {
}
```

**Comment by chrismair** [comment]
For what it's worth, I was not able to reproduce this issue if the annotation had no parameters, e.g. `@AnAnnotation`, but I was able to see the problem with: 
```
        @AnAnnotation("abc")
        // some comment
        void aMethod() {
        }
```

**Comment by chrismair** [comment]
I didn't fully fix the issue of annotations+comments messing up the line numbers, but I did add a check so that it prevents the false violation if it cannot find the proper line, so that fixes this particular issue.

Available in CodeNarc 1.3.
